src/api/initialDataTypes.js

```diff
@@ -74,53 +74,197 @@ export type AvailableVideoChatProviders = $ReadOnly<{|
 |}>;
 
 export type InitialDataRealm = $ReadOnly<{|
```

Comment:
In the first commit, is the idea to catch up completely with the current list in the API docs? I think doing so would be good -- it'd help make it more tractable to review, and would also give us a good starting point for incremental updates from here. (Like "this is current to feature level 107", and then we can read the [changelog](https://zulip.com/api/changelog) starting from there to make a future update.)

Currently it isn't -- e.g. it doesn't have `max_stream_name_length`. (I searched on https://zulip.com/api/register-queue#response for "present if realm is", and that was the first result.)

Reply:
Eep, yes, I definitely meant to get all 82 matches for "present if realm is" there; looks like I missed at least the first few. I'm checking again now.